### ISSUE TITLE
Cut new release of tokio-threadpool

### ISSUE THREAD

**Issue by jonhoo** [tokio-rs/tokio#487]
Can we cut a new minor `tokio-threadpool` release for `rand 0.5`? Pretty low priority, but most of the ecosystem seems to move on, and the update is already on `master`.

**Comment by tobz** [comment]
@carllerche Ping on this.

**Comment by carllerche** [comment]
Dup of #520. Releasing is blocked on investigating some deadlock bugs.
